### DISCUSSION TITLE
常见问题说明

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#481] [Announcements/公告]
### 1. _提示某文件无法访问_、_文件不能创建_、_配置不能读取_、_日志打不开_、_除了收能量其他功能正常_、_启动卡死/白屏_。可以尝试以下两种方式(**二选一**)
  > 重启支付宝的这步很重要，请不要忽略。另外请先重新给予存储授权(更新后有时显示授权了，但是实际未授权)。
  - 尝试在sdcard/Download目录下创建useMedia文件夹，**重启支付宝**，观察sdcard/Android/media目录下是否存在com.eg.android.AlipayGphone文件夹，如果存在则切换正常，再打开芝麻粒进行配置。
  - 删除sdcard/Download/xqe文件夹，**重启支付宝**，观察sdcard/Download/xqe文件夹存在之后，再打开芝麻粒进行配置。

**Comment by soochowgreen** [comment]
按照您的说明，我在 sdcard/Download目录下创建useMedia文件夹，重启支付宝，观察sdcard/Android/media目录下存在com.eg.android.AlipayGphone文件夹后，该目录存在，说明是正常的，然后打开芝麻粒进行配置。

删除sdcard/Download/xqe文件夹，重启支付宝， 发现sdcard/Download/xqe文件夹一直不存在，但是芝麻粒中可以读到日志，那么日志保存到哪个文件夹里去呢？谢谢。

**Reply by soochowgreen** [reply]
知道了，存放到 com.eg.android.AlipayGphone/xqe 目录下了。
